src/disjoint_set.jl

```diff
@@ -57,22 +57,28 @@ function union!(s::IntDisjointSets, x::Integer, y::Integer)
     parents = s.parents
```

Comment:
Thanks for the quick reply. I think `parents` is used in `find_root_impl!` below.